app/components/paths/select_button_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if current_user.on_path?(path) %>
+  <%= button_to path_url(path), method: :get, class: "button button--primary #{classes}", data: { test_id: "#{path.title.downcase}-resume-path-btn" } do %>
```

Comment:
It's a testing convention borrowed from [JS](https://www.educative.io/answers/what-is-the-data-testid-attribute-in-testing). We use it all over the app 😄 

The reasoning behind it is the text content and/or classes and id attributes on elements are liable to change. So instead of relying on them to select dom elements in end-to-end tests. We give elements a dedicated test_id attribute which is only used for the test suite.

Reply:
Having had to fix capybara tests when someone changed the copy more than once, this is a really neat idea :laughing: